scaleway/resource_mnq_credential_test.go

```diff
@@ -0,0 +1,159 @@
+package scaleway
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	mnq "github.com/scaleway/scaleway-sdk-go/api/mnq/v1alpha1"
+)
+
+func TestAccScalewayMNQCreeds_Basic(t *testing.T) {
+	tt := NewTestTools(t)
+	defer tt.Cleanup()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: tt.ProviderFactories,
+		CheckDestroy:      testAccCheckScalewayMNQnNamespaceCreedDestroy(tt),
+		Steps: []resource.TestStep{
+			{
+				Config: `
+					resource "scaleway_mnq_namespace" "main" {
```

Comment:
We should test nats credentials with nats provider if possible

Reply:
While the provider has been found to be compatible with nats and is performing well, it may not be a suitable fit for the current test setup.